src/item.cpp

```diff
@@ -1744,7 +1766,7 @@ std::string item::tname( unsigned int quantity, bool with_prefix ) const
     }
 
     if (has_flag("FIT")) {
-        ret << _(" (fits)");
+        ret << _(" (fits) ");
```

Comment:
That was for readability, but yea, it never occurred to me to just add a space before the (P) like the rest. Will change. I'm not sure what change at 1634 you're talking about, but it could have been from me messing around trying to find a good place to add this. I've been learning by trial and error.